lib/govuk_docker/setup/dnsmasq.rb

```diff
@@ -44,7 +44,8 @@ def install_dnsmasq
   def configure_etc_resolver_devgovuk
     write_file(
       "/etc/resolver/dev.gov.uk",
-      "nameserver 127.0.0.1\nport 53"
+      File.read(GovukDocker::Paths::dnsmasq_conf),
```

Comment:
Much better! I didn't realise you could use `::` to refer to a class method! I think it's more conventional to use `.`, though.

Reply:
You can only do it if you specify `self` before the method's definition.

Thought the other methods were being called with `::` but have double-checked and they're not - will update to `.` 👍